src/test/java/org/aist/aide/mappingservice/integration/MappingServiceIntegrationTests.java

```diff
@@ -0,0 +1,147 @@
+package org.aist.aide.mappingservice.integration;
+
+import static org.aist.aide.mappingservice.utils.TestsConstants.*;
+
+import java.util.List;
+import org.aist.aide.mappingservice.domain.models.Mapping;
+import org.aist.aide.mappingservice.service.controllers.MappingController;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.junit4.SpringRunner;
+
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+@ActiveProfiles(profiles = "test")
+public class MappingServiceIntegrationTests {
+
+    @Autowired
+    private MappingController mappingController;
+
+    @After
+    public void tearDown() {
+        mappingController.deleteMapping(1);
```

Comment:
Won't these keys be different after each test? Are they not auto-incrementing?

Reply:
Oh, yeah I shouldnt have added that. I did that at one point checking. something. In the end I have no teardown.